README.md

```diff
@@ -300,15 +352,62 @@ This approach only makes sense for very big or tree-like structs.
 Only test unexported funcs if you can't access a path via exported funcs.
 Since they are unexported, they are prone to change.
 
+### Add examples to your test files to demonstrate usage
+```go
+func ExamleSomeInterface_SomeMethod(){
+	instance := New()
+	result, err := instance.SomeMethod()
+	fmt.Println(result, err)
+	// Output: someResult, <nil>
+}
+```
+
 ## Use linters
 
-Use linters (e.g. [gometalinter](https://github.com/alecthomas/gometalinter)) to
+Use [gometalinter](https://github.com/alecthomas/gometalinter) to
 lint your projects before committing.
```

Comment:
Maybe add a note about which linters to use

Reply:
add "all of them" and installation and usage instructions since they are not very clear on the gometalinter page